README.md

```diff
@@ -525,6 +525,38 @@ or with a light image:
 docker run -v /path/to/models:/models ghcr.io/ggerganov/llama.cpp:light -m /models/7B/ggml-model-q4_0.bin -p "Building a website can be done in 10 simple steps:" -n 512
 ```
 
+### Docker With CUDA
+
+Assuming one has the [nvidia-container-toolkit](https://github.com/NVIDIA/nvidia-container-toolkit) properly installed on Linux, or is using a GPU enabled cloud, `cuBLAS` should be accessible inside the container.
+
+#### Building Locally
+
+```bash
+docker build -t local/llama.cpp:full-cuda -f .devops/full-cuda.Dockerfile .
+docker build -t local/llama.cpp:light-cuda -f .devops/main-cuda.Dockerfile .
```

Comment:
```suggestion
docker build -t local/llama.cpp:main-cuda -f .devops/main-cuda.Dockerfile .
```

Reply:
In general, I'm looking to keep it in-line with the `docker` documentation upstream. This is true of the examples and the image names.

I think those things should be changed everywhere, but either upstream first and then with this change, or in both places if this lands first.